$OpenBSD: patch-data_glade_moserial_glade,v 1.1.1.1 2011/07/04 10:00:38 jasper Exp $

Remove Linux device names in favor of what BSD uses.

--- data/glade/moserial.glade.orig	Sat Jul  2 10:22:02 2011
+++ data/glade/moserial.glade	Sat Jul  2 10:22:41 2011
@@ -1127,14 +1127,14 @@ Space</property>
 	      <child>
 		<widget class="GtkComboBoxEntry" id="settings_device">
 		  <property name="visible">True</property>
-		  <property name="items" translatable="yes">/dev/ttyS0
-/dev/ttyS1
-/dev/ttyS2
-/dev/ttyS3
-/dev/ttyUSB0
-/dev/ttyUSB1
-/dev/ttyUSB2
-/dev/ttyUSB3</property>
+		  <property name="items" translatable="yes">/dev/cua00
+/dev/cua01
+/dev/cua02
+/dev/cua03
+/dev/cuaU0
+/dev/cuaU1
+/dev/cuaU2
+/dev/cuaU3</property>
 		  <property name="add_tearoffs">False</property>
 		  <property name="has_frame">True</property>
 		  <property name="focus_on_click">True</property>
